example/package.json

```diff
@@ -13,26 +13,29 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
+    "@shopify/react-native-skia": "0.1.196",
+    "@types/react": "~18.2.14",
     "bumbag": "^2.2.1",
     "bumbag-native": "^2.3.0",
-    "expo": "^48.0.0",
-    "expo-haptics": "~12.2.1",
-    "expo-splash-screen": "~0.18.2",
+    "expo": "^49.0.0",
+    "expo-haptics": "~12.4.0",
+    "expo-splash-screen": "~0.20.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-native": "0.71.8",
-    "react-native-gesture-handler": "~2.9.0",
-    "react-native-reanimated": "~2.14.4",
-    "react-native-svg": "13.4.0",
-    "react-native-web": "~0.18.11"
+    "react-native": "0.72.3",
+    "react-native-gesture-handler": "~2.12.0",
+    "react-native-reanimated": "^3.3.0",
+    "react-native-svg": "13.9.0",
+    "react-native-web": "~0.19.6",
+    "typescript": "^5.1.3"
```

Comment:
It might be good to break all of the expo 49 and expo package updates into a separate PR so changes can be isolated. Quite a lot has been updated here. How much of is it related to the reanimated 3.x upgrade?